source/administration/object-management/create-lifecycle-management-expiration-rule.rst

```diff
@@ -110,3 +118,21 @@ delete markers:
   which to expire noncurrent object versions. For example, specify ``30d`` to
   expire a version after it has been noncurrent for at least 30 days.
 
+Expire All Versions of a Deleted Object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting with :mc-release:`MinIO Server RELEASE.2024-05-01T01-11-10Z`, MinIO supports deleting all versions of an object that has a delete marker as its latest version.
+MinIO only supports this function with JSON.
+
+To add this function, first export the rule to modify with :mc:`mc ilm rule export`.
```

Comment:
Maybe `function` was intended to be `functionality`? Could also say `To add this rule`

Reply:
I was trying to avoid so many instances of `rule` in the line, as that would make it three occurrences.
Went with a different workaround instead while avoiding the use of `function` here and on line 125.